app/helpers/value_helper.rb

```diff
@@ -69,9 +69,14 @@ def links_to_collections(args)
     end.join('<br>').html_safe
   end
 
+  # Discussed removing this method because the field should not be modified.
   def value_for_originInfo_date_created_tesim(args)
-    val = Time.parse(args[:document][args[:field]].first)
+    field_value = args[:document][args[:field]].first
+    # Try to parse the string as a time value
+    val = Time.parse(field_value)
```

Comment:
Do not use `Time.parse` without zone. Use one of `Time.zone.parse`, `Time.parse.current`, `Time.parse.in_time_zone`, `Time.parse.utc`, `Time.parse.getlocal`, `Time.parse.iso8601`, `Time.parse.jisx0301`, `Time.parse.rfc3339`, `Time.parse.to_i`, `Time.parse.to_f` instead.